root/static/scripts/tests/guessFeat.js

```diff
@@ -12,418 +12,451 @@ import test from 'tape';
-                    names: [
-                        {name: 'Ill', joinPhrase: ' feat. '},
-                        {name: 'Hfi', joinPhrase: ', '},
-                        {name: 'Oxmo', joinPhrase: ' et '},
-                        {name: 'Pit Baccardi', joinPhrase: ''},
-                    ],
-                },
-            }
+      },
+    },
+    {
+      input: {
+        name: '50,000 ft.',
+        artistCredit: {names: [{name: 'The Hang Ups', joinPhrase: ''}]},
+      },
+      // no change
+      output: {
+        name: '50,000 ft.',
+        artistCredit: {names: [{name: 'The Hang Ups', joinPhrase: ''}]},
+      },
+    },
+    {
+      input: {
+        name: 'Classe x et freestyle (Feat. Hfi, Oxmo et Pit Baccardi)',
+        artistCredit: {names: [{name: 'Ill', joinPhrase: ''}]},
+      },
+      output: {
+        name: 'Classe x et freestyle',
+        artistCredit: {
+          names: [
+            {name: 'Ill', joinPhrase: ' feat. '},
+            {name: 'Hfi', joinPhrase: ', '},
+            {name: 'Oxmo', joinPhrase: ' et '},
+            {name: 'Pit Baccardi', joinPhrase: ''},
+          ],
         },
-        {
-            input: {
-                name: 'The Lion (Featuring Sphere720 And Tariq)',
-                artistCredit: {names: [{name: 'Dragon Fli Empire', joinPhrase: ''}]},
-            },
-            output: {
-                name: 'The Lion',
-                artistCredit: {
-                    names: [
-                        {name: 'Dragon Fli Empire', joinPhrase: ' featuring '},
-                        {name: 'Sphere720', joinPhrase: ' And '},
-                        {name: 'Tariq', joinPhrase: ''},
-                    ],
-                },
-            }
+      },
+    },
+    {
+      input: {
+        name: 'The Lion (Featuring Sphere720 And Tariq)',
+        artistCredit: {names: [{name: 'Dragon Fli Empire', joinPhrase: ''}]},
+      },
+      output: {
+        name: 'The Lion',
+        artistCredit: {
+          names: [
+            {name: 'Dragon Fli Empire', joinPhrase: ' featuring '},
+            {name: 'Sphere720', joinPhrase: ' And '},
+            {name: 'Tariq', joinPhrase: ''},
+          ],
         },
-        {
-            input: {
-                name: 'The Bell (Paranoia Network Remix), featuring Pete Seeger & DJ Spooky',
-                artistCredit: {names: [{name: 'Stephan Smith', joinPhrase: ''}]},
-            },
-            output: {
-                name: 'The Bell (Paranoia Network Remix)',
-                artistCredit: {
-                    names: [
-                        {name: 'Stephan Smith', joinPhrase: ' featuring '},
-                        {name: 'Pete Seeger', joinPhrase: ' & '},
-                        {name: 'DJ Spooky', joinPhrase: ''},
-                    ],
-                },
-            }
+      },
+    },
+    {
+      input: {
+        name: `The Bell (Paranoia Network Remix),
+               featuring Pete Seeger & DJ Spooky`,
```

Comment:
Technically using template strings here changes the actual test by adding newlines and whitespace to the input string, e.g. now this is:
```
"The Bell (Paranoia Network Remix),\n                 featuring Pete Seeger & DJ Spooky"
```
But if the tests still pass I guess it's fine (at least we know it handles whitespace correctly).

Reply:
Turned it to concatenated strings to make it more similar to the names "in the wild"